archive/j/java/LeapYear.java

```diff
@@ -0,0 +1,21 @@
+import java.util.*;
```

Comment:
Awesome program, but we currently don't support it. Any chance you could write up a project spec for us? [See projects](https://sample-programs.therenegadecoder.com/projects/).